components/BaseLinePlotComponent.vue

```diff
@@ -152,6 +181,7 @@ export default class BaseLinePlotComponent extends Vue {
     let out = [];
     for (let i = 0; i < this.timesWithPositiveIds.length; ++i) {
       if (this.timesWithPositiveIds[i]['@_id'] == this.currentAttempt?.['@_id']) out.push(6);
+      if (this.timesWithPositiveIds[i]['@_id'] == this.medianAttempt?.['@_id']) out.push(6);
```

Comment:
You're missing an `else` which can offset everything by pushing twice in the array
```suggestion
      else if (this.timesWithPositiveIds[i]['@_id'] == this.medianAttempt?.['@_id']) out.push(6);
```